app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (-1, -7, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
```

Comment:
Add test cases to cover situations when `cat_age` and `dog_age` have different incoming values as mentioned [in the checklist](https://github.com/nsdNite/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).